src/Http/Event/ImportEventRequestHandler.php

```diff
@@ -132,7 +132,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $eventAdapter = new Udb3ModelToLegacyEventAdapter($event);
 
         $mainLanguage = $eventAdapter->getMainLanguage();
-        $title = new Title($eventAdapter->getTitle()->toString());
+        $title = $event->getTitle()->getTranslation($event->getTitle()->getOriginalLanguage());
```

Comment:
For a follow-up pull request:

I think we can improve the `TranslatedValueObject` and give it an extra helper method `getOriginalValue`